app/src/main/java/org/sopt/and/navigation/BottomNavigationItem.kt

```diff
@@ -0,0 +1,30 @@
+package org.sopt.and.navigation
+
+import org.sopt.and.R
+
+
+data class BottomNavigationItem(
+    val label : Int = R.string.home,
+    val icon : Int = R.drawable.ic_home,
```

Comment:
Android Resource의 id를 필요로하는 타입에 대해서는
앞에 @StringRes, @DrawableRes 등등의 Annotation을 붙여주어 resource id를 필요로 함을 확실하게 명시해줄 수 있습니다 !!

```suggestion
    @StringRes val label : Int = R.string.home,
    @DrawableRes val icon : Int = R.drawable.ic_home,
```

Reply:
몰랐던 부분이네요 감사합니다!